### DISCUSSION TITLE
example or guide to expand image

### DISCUSSION THREAD

**Discussion by tomdavidson** [Acly/krita-ai-diffusion#951] [Q&A]
Im struggling understanding how to accomplish the goal of expanding and image of a painting an additional 190 px  on all sides. Can anyone direct me to a tutorial or some other how to?

**Comment by Drakosha405** [comment]
![1](https://github.com/user-attachments/assets/fc1e41c8-3299-4e59-9081-6a1769ad1285)

**Reply by sraney** [reply]
Using expansion via section.   You can also do a bit of color and shape guidance on the new area also.  

https://github.com/user-attachments/assets/f1cb35d3-4998-4147-87f7-a547df6f64f1

**Reply by tomdavidson** [reply]
This is so kool! Thank you for the help, I waited until my wired internet connection was fixed to try it out. (9 yr bashed my fiber in half with some rocks to see what the insides looked like :D )

is the edge in this screen shot more of a stable diffusion issue ?

![image](https://github.com/user-attachments/assets/2d4841ef-f762-4fa0-8177-ad3ed534d35c)

**Comment by mastertheos-dev** [comment]
I can't use Expand Image with plugin 1.40.1. I get either black borders or, for example, the head hair is on a black border, but not the background of  image.
